### DISCUSSION TITLE
Integrating NVDA with Chromium

### DISCUSSION THREAD

**Discussion by kyrylo-pshenychnyi** [nvaccess/nvda#14532] [Q&A] [answered]
Hi everyone, 

I am trying to integrate NVDA with my app that uses Chromium browser for the visual representation. Right now, the Chromium accessibility tree is visible from my app by Accessibility Insights for Windows app. However, NVDA does not narrate focused elements (buttons, input fields, etc.). 

I build NVDA to see what's going on - to be precise I see that the `get_accDescription` function is not called for a focused node. So my question is - how can I debug NVDA to see why this function is not called? 

Any help will be appreciated.

**Comment by seanbudd** [comment, marked as answer]
NVDA relies on a few window class names to determine something is a chromium document.
`Chrome_RenderWidgetHostHWND`, `Chrome_WidgetWin_0`, `Chrome_WidgetWin_1`.
This may be related. There was a similar discussion here #14423

**Comment by josephsl** [comment]
Hi, are we talking about Chromium as a web browser or a web view? Thanks.

**Reply by kyrylo-pshenychnyi** [reply]
> Hi, are we talking about Chromium as a web browser or a web view? Thanks.

Hi, we are talking about Chromium as a web view. 

I managed to solve my issue using @seanbudd answer and this discussion https://github.com/nvaccess/nvda/issues/14423
